sourcetransaction.go

```diff
@@ -33,12 +33,12 @@ type SourceTransaction struct {
 // source transaction.
 func (t *SourceTransaction) UnmarshalJSON(data []byte) error {
 	type sourceTransaction SourceTransaction
-	var st sourceTransaction
-	err := json.Unmarshal(data, &st)
+	var v sourceTransaction
```

Comment:
guessing it's expected that this can not just be an ID right?

Reply:
Yeah exactly. It was like this before, so I just left it.